### ISSUE TITLE
Once I present a new vc, the animation doesn't perform in the correct way

### ISSUE THREAD

**Issue by magnaroksjj** [Ramotion/folding-cell#139]
Everything performs well as I test in this own TableView. Once I present a new VC and go back to the TableView, the animation of folding-cell doesn't perform in the correct way any longer.

**Comment by magnaroksjj** [comment]
I tested my app in a real phone and found it just worked correctly, but slow and not fluently . The more I present and dismiss a new VC, the slower the animation behaves.
I have not learned programming for a long time(Swift is my first programming language to learn by my self), what I can suspect is that there may be a problem with memory. Then, I used Instruments-Leaks to check my app. The result showed no leaks at all.
Now, I am confused where the problem is but with no clue.
